packages/scripts/scripts/lint-md-docs.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+const { sync: spawn } = require( 'cross-spawn' );
+const { sync: resolveBin } = require( 'resolve-bin' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	fromConfigRoot,
+	getArgsFromCLI,
+	hasArgInCLI,
+	hasFileArgInCLI,
+	hasProjectFile,
+} = require( '../utils' );
+
+const args = getArgsFromCLI();
+
+const defaultFilesArgs = hasFileArgInCLI() ? [] : [ '**/*.md' ];
+
+// See: https://eslint.org/docs/user-guide/configuring#using-configuration-files-1.
```

Comment:
URLs should point to https://github.com/igorshubovych/markdownlint-cli :)

Reply:
Copy paste miss.